src/components/nav/Nav.js

```diff
@@ -1,3 +1,169 @@
-const Nav = () => {};
+import React, { useRef, useState } from 'react';
+import styled from 'styled-components';
+import { GiHamburgerMenu } from 'react-icons/gi';
+import { IoIosSearch } from 'react-icons/io';
+import DropMenu from './DropMenu.js';
+import { useNavigate } from 'react-router-dom';
+
+const Nav = () => {
+  const [dropMenu, setDropMenu] = useState(false);
+  const navigate = useNavigate();
+  const outSection = useRef();
+
+  //로그인 토큰 받으면 텍스트 변경
+  const [isLogin, setIsLogin] = useState(false);
+  const wantToken = localStorage.getItem('want_token');
+
+  const showDropMenu = () => {
+    setDropMenu(!dropMenu);
+  };
+
+  // 메인으로 보내기 (wantUS 클릭 시)
+  // const goToMain = () => {
+  //   navigate('/main');
+  // };
+
+  const checkLogin = () => {
+    wantToken ? setIsLogin(!isLogin) : navigate('login');
+  };
+
+  const closeDropMenu = e => {
+    if (outSection.current === e.target) {
+      setDropMenu(false);
+    }
+  };
+
+  return (
+    <>
+      <NavWrapper ref={outSection} onClick={closeDropMenu}>
+        <NavContainer>
+          <Mainbar>
+            <HamburgerBar onClick={showDropMenu}>
+              <GiHamburgerMenu />
+            </HamburgerBar>
+            <Title>wantUS</Title>
+          </Mainbar>
+          <MenuListContainer>
+            {GLOBAL_NAV.map(({ content, id }) => {
+              return <MenuList key={id}>{content}</MenuList>;
+            })}
+          </MenuListContainer>
+          <NavRightContainer>
+            <UserSection>
+              <SearchIcon />
+              {isLogin ? (
+                <Login onClick={checkLogin}>로그아웃</Login>
+              ) : (
+                <Login onClick={checkLogin}>회원가입/로그인</Login>
+              )}
+            </UserSection>
+            <CompanyService>기업 서비스</CompanyService>
+          </NavRightContainer>
+        </NavContainer>
+      </NavWrapper>
+      {dropMenu ? <DropMenu /> : null}
```

Comment:
```suggestion
      {dropMenu && <DropMenu /> }
```
false를 null로 반환한다면 && 단축평가를 사용할 수 있습니다.